config/config.go

```diff
@@ -43,6 +44,7 @@ type BlockManagerConfig struct {
 	// DAStartHeight allows skipping first DAStartHeight-1 blocks when querying for blocks.
 	DAStartHeight uint64            `mapstructure:"da_start_height"`
 	NamespaceID   types.NamespaceID `mapstructure:"namespace_id"`
+	FraudProofs   bool              `mapstructure:"fraud_proofs"`
```

Comment:
```suggestion
	StateFraudProofs   bool              `mapstructure:"fraud_proofs"`
```
I would change FraudProof to StateFraudProof as we might have different fraud proofs in the future. If you accept the change, please change it also in every other context like flagname, log message etc.

Reply:
agree, but this can be delayed to until when we have other fraud proofs? asking this because: 1) the fraud proof to state fraud proof terminology change may be required to be done across multiple repos? 2) the change will touch lot of code from @Manav-Aggarwal (basically doing nuance changes)?
if so, we can do this name migration in a separate issue? i understand that, it might increase the technical debt when more fraud proof code gets added, will be happy to hear further suggestion.

Reply:
Agree, we can create a separate issue that can have a lower priority to rename fraud proof to specifically be state fraud proof in case we add more fraud proofs in the future.

Reply:
Ok makes sense to me I created a seperate issue #637